### PR TITLE
Update prebuilt dependencies

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22689/HighFidelity-Beta-PR16536-0240a6e.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22689/HighFidelity-Quest-Beta-PR16536-0240a6e.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22689/HighFidelity-Beta-PR16536-0240a6e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22689/HighFidelity-Beta-Interface-PR16536-0240a6e.dmg

**Comment by hd23408** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22693/HighFidelity-Beta-PR16536-0240a6e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22693/HighFidelity-Beta-Interface-PR16536-0240a6e.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22693/HighFidelity-Beta-PR16536-0240a6e.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22693/HighFidelity-Quest-Beta-PR16536-0240a6e.apk)

File: cmake/macros/TargetSRanipalEye.cmake [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#
+macro(TARGET_SRANIPAL)
+
+if (WIN32)
+    target_include_directories(${TARGET_NAME} SYSTEM PUBLIC "${VCPKG_INSTALL_ROOT}/include")
+    find_library(SRANIPAL_LIBRARY NAMES SRanipal PATHS ${VCPKG_INSTALL_ROOT}/lib/ NO_DEFAULT_PATH)
+    target_link_libraries(${TARGET_NAME} ${SRANIPAL_LIBRARY})
+
+    find_library(SRANIPAL_LIBRARY NAMES SRanipal PATHS ${VCPKG_INSTALL_ROOT}/lib/ NO_DEFAULT_PATH)
+    target_link_libraries(${TARGET_NAME} ${SRANIPAL_LIBRARY})
```
Comment (SimonWalton-HiFi) [inline cmake/macros/TargetSRanipalEye.cmake]: Is there a reason for having the find and target lines twice?
Also, it's a little inconsistent with the names SRanipal/SRanipalEye.

**Pull request merged**